src/components/Sing/SequencerRuler.vue

```diff
@@ -93,6 +107,7 @@ const endTicks = computed(() => {
 const width = computed(() => {
   return tickToBaseX(endTicks.value, tpqn.value) * zoomX.value;
 });
+// measureInfosの計算を調整
 const measureInfos = computed(() => {
```

Comment:
このコメントの意図がわからないかも･･･？　なくても大丈夫そう？
```suggestion
const measureInfos = computed(() => {
```

Reply:
@Hiroshiba 
こちら修正しました！

あわせてここも`while`ループになっていたため`map`に変更しました